superset/sql_lab.py

```diff
@@ -231,11 +231,12 @@ def handle_error(msg):
 
     # check whether the result set is comprised of lists or dict
     if data and len(data) > 0:
```

Comment:
if data:

Reply:
Yep you're right the len check is redundant